packages/deb/bionic/kubelet/debian/control

```diff
@@ -11,8 +11,5 @@ Vcs-Browser: https://github.com/kubernetes/kubernetes
 Package: kubelet
 Architecture: {{ .DebArch }}
 Depends: iptables (>= 1.4.21), iproute2, socat, util-linux, mount, ebtables, ethtool, conntrack, ${misc:Depends}
-Provides: kubernetes-cni
-Conflicts: kubernetes-cni
-Replaces: kubernetes-cni
```

Comment:
Should we change deb spec since deb packages work correctly?

What is going to happen when the kubelet package gets upgraded? If we drop this, I'd suppose that the `kubernetes-cni` package will stay in place, which may cause issues. But since I'm not an expert with this, someone who has experience should confirm.

Reply:
Did we verify that the debs work as intended? I thought the same issue exists on debs and rpms? cc @cpanato can you confirm?

Reply:
@saschagrunert Personally, I had no problem provisioning 1.18.3 and 1.18.4 clusters, same for 1.17.

Reply:
Okay so we may revert that change. 👍